automation/package.json

```diff
@@ -1,26 +1,26 @@
 {
-    "name": "webdriverio-tests",
-    "version": "0.1.0",
-    "private": true,
-    "devDependencies": {
-        "@wdio/allure-reporter": "^7.16.13",
-        "@wdio/cli": "^7.16.12",
-        "@wdio/cucumber-framework": "^7.16.12",
-        "@wdio/local-runner": "^7.16.12",
-        "@wdio/spec-reporter": "^7.16.11",
-        "@wdio/sync": "^7.16.13",
-        "chromedriver": "^99.0.0",
-        "wdio-chromedriver-service": "^7.2.6"
-    },
-    "scripts": {
-        "wdio": "wdio run wdio.conf.js",
-        "wdio:local": "wdio run wdio.conf.local.js",
-        "wdio:local:test": "wdio run wdio.conf.local.js --cucumberOpts.tagExpression='@test'",
-        "yarn:start": "cd .. && yarn start",
-        "yarn:install:deps": "cd .. && yarn"
-    },
-    "dependencies": {
-        "@wdio/browserstack-service": "^7.16.13",
-        "allure-commandline": "^2.17.2"
-    }
+  "name": "webdriverio-tests",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@wdio/allure-reporter": "^7.16.13",
+    "@wdio/cli": "^7.16.12",
+    "@wdio/cucumber-framework": "^7.16.12",
+    "@wdio/local-runner": "^7.16.12",
+    "@wdio/spec-reporter": "^7.16.11",
+    "@wdio/sync": "^7.16.13",
+    "chromedriver": "^99.0.0",
+    "wdio-chromedriver-service": "^7.2.6"
+  },
+  "scripts": {
+    "wdio": "wdio run wdio.conf.js",
+    "wdio:local": "wdio run wdio.conf.local.js",
+    "wdio:local:test": "wdio run wdio.conf.local.js --cucumberOpts.tagExpression='@test'",
+    "yarn:start": "cd .. && yarn start",
+    "yarn:install:deps": "cd .. && yarn"
```

Comment:
follow up (non-blocking): can you please ticket the following:

* This script is pointless and vanity they should be removed and the workflow should just inline the commands
* This package is using NPM and should be moved to using yarn

Reply:
Yes but:
1. This project is not using yarn so exposing it as a dependency in this package JSON does not make sense
2. This is a vanity script and could just be inline into the workflow file without the need for these
3. These are not doing anything complex (2 commands) so I would expect these to be run in line in the workflow file. If they were calling more things then that would be fine

So can we add a ticket to tidy this up please, it will only take 5 mins to do :+1: